API_Guidelines.md

```diff
@@ -72,49 +72,42 @@ Payload
 
 ## PUT
 
-> `PUT` requests update records that already exist with new information
+> `PUT` requests update records that already exist with new or updated information
 
-### Example
-
-#### Current model:
-```javascript
-{
-    "bands": {
-        "id": 8,
-        "name": "Radiohead",
-        "bestAlbum": "OK Computer",
-        "bestBandEver": false
-    }
-}
-```
+### Request
+URL
+:   `apiHost.com/movies/2`
 
-#### Update is desired:
+Ember Data Method
+:   
 ```javascript
-set(model, 'bestBandEver', true);
-model.save(); // This initializes the PUT request
+//lookup record in the local store
+let movie = this.store.findRecord('movie', 2); // returns record of {"id": 2, "title": "Goodfellas", "year": "1990"}
```

Comment:
i disagree. this is more about how to use the api for the framework, not our standard for writing this method. if you look at the official ember guides, they also do not destructure